wallets/leapextension.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "../wallet.schema.json",
+  "wallet_name": "leapextension",
+  "pretty_name": "Leap Cosmos Wallet",
+  "website": "https://www.leapwallet.io/",
+  "git_repo": "",
+  "supported_chains": [
+    "cosmoshub",
+    "juno",
+    "osmosis",
+    "secretnetwork",
+    "agoric",
+    "akash",
+    "assetmantle",
+    "axelar",
+    "bitsong",
+    "cheqd",
+    "chihuahua",
+    "comdex",
+    "desmos",
+    "emoney",
+    "evmos",
+    "fetchhub",
+    "gravitybridge",
+    "injective",
+    "irisnet",
+    "jackal",
+    "kujira",
+    "likecoin",
+    "mars",
+    "persistence",
+    "atlantic",
+    "sifchain",
+    "sommelier",
+    "stargaze",
+    "starname",
+    "stride",
+    "teritori",
+    "umee"
+  ],
+  "features": ["suggest_chain"],
```

Comment:
Do you have any documentation on the suggest chain feature? Would like to learn more about how users can add chain to their wallet

Reply:
Here is the link to the suggest chain docs.
https://docs.leapwallet.io/cosmos/connect-to-leap-on-dapps/suggest-chain